baybe/surrogates/base.py

```diff
@@ -83,6 +86,34 @@ def to_botorch(self) -> Model:
 
         return AdapterModel(self)
 
+    @staticmethod
+    def _get_parameter_scaler(
+        parameter: Parameter,
+    ) -> ScalerProtocol | Literal["passthrough"]:  # noqa: F821
+        """Return the scaling method to be used for the given parameter."""
+        return MinMaxScaler()
+
+    def _make_input_scaler(
+        self, searchspace: SearchSpace, measurements: pd.DataFrame
+    ) -> ColumnTransformer:
+        """Make a scaler to be used for transforming computational dataframes."""
+        from sklearn.compose import make_column_transformer
+
+        # Create the composite scaler from the parameter-wise scaler objects
+        # TODO: Filter down to columns that actually remain in the comp rep of the
+        #   searchspace, since the transformer can break down otherwise.
+        transformers = [
+            (self._get_parameter_scaler(p), p.comp_df.columns)
```

Comment:
Technically, a parameter scaler is something that scales a parameter. Intuitively, this would mean for me that such a scaler and such a call should be part of a `Parameter´-related class, not the surrogate class. Knowing you, I am sure that you have thought about this as an alternative, and I would hence be interested in hearing your arguments for this design here - I guess mainly because "You would never scale a parameter without using it then in a surrogate, so it should be part of the surrogate's logic, not the parameter's logic" (which is an argument I agree with, I just want to know if there are other aspects that I do not see)?